riak_test/log_handoff_test.erl

```diff
@@ -75,6 +75,7 @@ test_handoff(RootNode, NewNode, NTestItems) ->
     %% added to the root node.  Note: systest_read() returns
     %% /non-matching/ items, so getting nothing back is good:
     lager:info("Validating data after handoff:"),
+    timer:sleep(100),
```

Comment:
"passed on my local machine" is not good enough.

You need to write some code that waits for the service to stop, and not rely on a hardcoded wait of a few milliseconds.